packages/next/build/webpack/config/blocks/css/loaders/helpers/SequentialCSSModuleLocalIdentGenerator.ts

```diff
@@ -0,0 +1,123 @@
+// Valid CSS class name's Character set
```

Comment:
Based on the disclaimer you added to the PR:

> This feature is based on the great work made by the people at Styletron, using a modified version of their MIT Licensed SequentialIDGenerator.

We should include the MIT License of the project in this file.

Reply:
Thanks for the reminder!